uiowa/uiowa
# Update media_entity_facebook, lazy, schema_metatag

Resolves #2361
Fixes #2397
Relates to #2282 

I have added the API id and secret to each environment's Acquia secrets file.  For example, you can ssh into uiowa.dev and then view the secrets.settings.php file to use the credentials locally in a site's local.settings.php file.

This patch is a short term fix: https://www.drupal.org/project/media_entity_facebook/issues/3176739#comment-13882401

Also snuck in a fix a lazy load regression by updating to the latest stable release, and an update to schema_metatag which removes a patch.

**Reference**:
https://docs.acquia.com/resource/secrets/

## To Test
- `composer install && blt ds --site=default` (no database update)
- Assuming you are in an environment that is setup to use the credentials, you should be able to add a new Facebook post via media without issue.
- A non-working setup will either error out or silently fail but fail in bringing over any metadata related to the post.
- You can also check to see that Instagram is also working as this is now served by the same variable.
- Since we aren't writing the credentials to the db, there should be no config export.
- Fade-in effect should be back for image loading. #2397
- On uiowa.local.drupal.uiowa.edu, make sure that SearchAction is still in the `application/ld+json` schema.org markup.
